js/scripts.js

```diff
@@ -8,5 +8,4 @@ list.addEventListener('click', function(e) {
 
 add.addEventListener('click', function(e) {
   list.innerHTML +='<li>item</li>';
```

Comment:
extend your text `item` with number. number you can get from var add. this variable is an Array so you can check it length :)